src/libraries/System.Runtime/tests/System/Type/TypeTests.cs

```diff
@@ -219,13 +1196,102 @@ public void MakeByRefType_Invoke_ReturnsExpected(Type t)
             Assert.Equal(tRef1, tRef2);
 
             Assert.True(tRef1.IsByRef);
+            Assert.False(tRef1.IsPointer);
```

Comment:
Nit, maybe slightly clearer to just have one local `tRef` and write `Assert.Equal(tRef, t.MakeByRefType());

Reply:
and elsewhere